nva-datamodel-java/src/main/java/no/unit/nva/model/contexttypes/MediaContribution.java

```diff
@@ -29,16 +31,20 @@ public class MediaContribution implements PublicationContext {
 
     @JsonProperty(PART_OF)
     private final SeriesEpisode partOf;
+    @JsonProperty(LINK)
+    private final URI link;
```

Comment:
Isn't this an associated artefact in reality?

Reply:
If the cristin post is a podcast and this links points to said podcast, the link is an associated artefact in my opinion.

Reply:
Fully possible. Do we place podcasts links, tv programs links in associatedArtifacts? If yes, i remove this pr.

Reply:
Do we use AssociatedArtifacts.associatedLink for that purpose? Isn't it mostly DOI? I will check with @brinxmat 

<img width="223" alt="image" src="https://github.com/BIBSYSDEV/nva-datamodel-java/assets/107109909/f328fa6d-32c9-44eb-9ead-4924f2419a4d">

Reply:
Associated artefact, yes.

Reply:
Fikk svar fra Rurik, du har rett. Fjerner pr.